docs/index.rst

```diff
@@ -173,7 +196,7 @@ log in view. (If the login view is not set, it will abort with a 401 error.)
 The name of the log in view can be set as `LoginManager.login_view`.
 For example::
 
-    login_manager.login_view = "users.login"
+    login_manager.login_view = "login"
```

Comment:
Why does this need changing?

Reply:
For consistency with the rest of the examples in this document.

From the PR:
>In a trivial Flask application (which most users will be writing in order to understand/follow along with the documentation), you are not likely to have views like users.login. In the documentation's own example, the name of the login view is login